autogpt_platform/frontend/orval.config.ts

```diff
@@ -45,23 +40,23 @@ export default defineConfig({
       afterAllFilesWrite: "prettier --write",
     },
   },
-  autogpt_zod_schema: {
-    input: {
-      target: `./src/app/api/openapi.json`,
-      override: {
-        transformer: "./src/app/api/transformers/fix-tags.mjs",
-      },
-    },
-    output: {
-      workspace: "./src/app/api",
-      target: `./__generated__/zod-schema`,
-      schemas: "./__generated__/models",
-      mode: "tags-split",
-      client: "zod",
-      indexFiles: false,
-    },
-    hooks: {
-      afterAllFilesWrite: "prettier --write",
-    },
-  },
```

Comment:
We can add the generated zod and msw files when we need them, for now keep it lean with what is just used on the app 💆🏽